README.md

```diff
@@ -27,23 +27,24 @@ This is the starting point for joining and contributing to the Elyra community -
 improving code, giving talks etc.
 
 
-### Dev Meetings
-Join us for our scrum to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
+### Dev Meetings Elyra General
+Join us to discuss development items you're working on or have questions about. Everyone is welcome and participation is optional. Note: Currently the Elyra community call is co-located with the CLAIMED community call.
 
-**When**: Every Friday at [8:30AM Pacific](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco&)
+**When**: Every Friday at [8:30-9:00 AM Pacific](https://dateful.com/convert/san-francisco-california?t=0830)
```

Comment:
@romeokienzler looks like US is already on summer / daylight savings time. Please double-check both times.
i.e. whether 16:30 or 17:30.